### ISSUE TITLE
Custom Keybindings within the ZIP

### ISSUE THREAD

**Issue by bphenriques** [schellingb/dosbox-pure#365]
Hello!

Is it possible to set a keybindings next to each game? I know I can set it up in the UI but I want the config to live next to the `.zip` file for portability across my systems.

Edit: more specifically, I have several games playable (thanks to the default useful keybindings!), however I find the keys disposition quite unusual (e.g., jumping with R2) and I would like to set the right keybindings (for me). I am aware of the `DOSBOX.CONF` file so it might be possible to leverage that feature if there is an API for keybindings.

**Comment by bphenriques** [comment]
Relates to https://github.com/schellingb/dosbox-pure/issues/360

**Comment by bphenriques** [comment]
I actually managed to find the mapper configuration inside the save file which, in theory, I can create a `mapperfile=mapper.map` entry inside the `DOSBOX.conf` file.

How portable is the mapper? Consider an example where I created one for a regular gamepad:
- Can I reuse it if I use another gamepad with the same layout? 
- Is it ignored if I use a keyboard that does not share the same layout as a regular gamepad?

This is tricky, if I manage I can contribute to the FAQ so that others can benefit
